developers.rst

```diff
@@ -1,8 +1,8 @@
-***************
-Developer guide
-***************
+**************
+Building Nengo 
```

Comment:
I think in the software context "building" usually refers to "compiling"/"installing", rather than "working on" (e.g., "building nengo from source").  Maybe "Developing Nengo"?

Reply:
Right, this was also a tricky decision.  Here's my thinking: the term 'developer' was causing confusion in places, so 'contributor' seemed like a good alternative.  However, 'contributor' was already being used as a subsection (ie for software writers, not reviewers, etc.), and is used throughout the website with that meaning.  So, i picked 'building nengo' to try to be neutral to whether it was for software writers, or others.  But you're right 'build=compile' is pretty common in sw.  Maybe the best alternative is to dig in and change all 'contributor' to 'developer' and rename this 'contributor guide'...  lmk

Reply:
i actually came to the same conclusion and am doing that now :)